src/components/drop_down/drop_down.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { DropDown } from './index'
+
+describe('DropDown', () => {
```

Comment:
This is going to need tests to make sure that it renders the correct component when a link is clicked